src/pandapipes/create.py

```diff
@@ -439,8 +437,8 @@ def create_pipe(net, from_junction, to_junction, std_type, length_km, k_mm=0.2,
     pipe_parameter = load_std_type(net, std_type, "pipe")
```

Comment:
Could we add a warning here, if "alpha_w_per_m2k" is contained in the kwargs? Just to make the change more transparent.